core-bundle/contao/widgets/FileTree.php

```diff
@@ -415,17 +415,14 @@ protected function getPreviewImage(File $objFile, $strInfo, $blnRemovable = fals
 
 			$img = $picture->getImg($projectDir, $container->get('contao.assets.files_context')->getStaticUrl());
 
-			$buffer = \sprintf('<img src="%s"%s width="%s" height="%s" alt="%s" loading="lazy" data-contao--tooltips-target="tooltip">', $img['src'], $img['srcset'] != $img['src'] ? ' srcset="' . $img['srcset'] . '"' : '', $img['width'], $img['height'], $strInfo);
+			$buffer = \sprintf('<img src="%s"%s width="%s" height="%s" alt="%s" class="%s" loading="lazy" data-contao--tooltips-target="tooltip">', $img['src'], $img['srcset'] != $img['src'] ? ' srcset="' . $img['srcset'] . '"' : '', $img['width'], $img['height'], $strInfo, $strClass);
 		}
 		else
 		{
 			$buffer = Image::getHtml('placeholder.svg', $strInfo, 'data-contao--tooltips-target="tooltip"');
```

Comment:
There is no need to pass the class here imo (placeholder).

Reply:
I do think the class still needs to be added here. While there is no CSS (anymore) for `.gimage`, the tooltip controller still uses it.